x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/bulk_create_suppressed_alerts_in_memory.ts

```diff
@@ -99,6 +132,89 @@ export const bulkCreateSuppressedAlertsInMemory = async ({
   return executeBulkCreateAlerts({
     suppressibleWrappedDocs,
     unsuppressibleWrappedDocs,
+    buildingBlockAlerts,
+    toReturn,
+    bulkCreate,
+    services,
+    ruleExecutionLogger,
+    tuple,
+    alertSuppression,
+    alertWithSuppression,
+    alertTimestampOverride,
+    experimentalFeatures,
+    maxNumberOfAlertsMultiplier,
+  });
+};
+
+/**
+ * wraps, bulk create and suppress alerts in memory, also takes care of missing fields logic.
+ * If parameter alertSuppression.missingFieldsStrategy configured not to be suppressed,
+ * regular alerts will be created for such events without suppression
+ */
+export const bulkCreateSuppressedSequencesInMemory = async ({
```

Comment:
I suppose I could. I like that it's next to the other functions that it uses, like executeBulkCreateAlerts though. If you think it will be cleaner to have it in another file I can move it.